### PR TITLE
Fixes #14070: Curl is not a rudder-server-root dependency

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/14070

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

File: rudder-webapp/SPECS/rudder-webapp.spec [hunk 1]
```diff
@@ -124,7 +124,7 @@ Obsoletes: ncf, ncf-api-virtualenv, rudder-techniques
 Conflicts: ncf, ncf-api-virtualenv, rudder-techniques
 
 # Dependencies
-Requires: rudder-inventory-ldap = %{real_epoch}:%{real_version}, rudder-server-relay = %{real_epoch}:%{real_version}, %{apache}, %{apache_tools}, git-core, rsync, openssl, %{ldap_clients}
+Requires: rudder-inventory-ldap = %{real_epoch}:%{real_version}, rudder-server-relay = %{real_epoch}:%{real_version}, %{apache}, %{apache_tools}, git-core, rsync, openssl, %{ldap_clients}, curl
```
Comment (peckpeck) [inline rudder-webapp/SPECS/rudder-webapp.spec]: what about debian ?

Comment (amousset) [reply rudder-webapp/SPECS/rudder-webapp.spec]: it is already there for debian

Comment (amousset) [reply rudder-webapp/SPECS/rudder-webapp.spec]: Oh no, indeed!

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**